notebooks/tutorials/umbrella_sampling.ipynb

```diff
@@ -0,0 +1,434 @@
+    "\n",
+    "# plot the histograms\n",
+    "metafilelines = []\n",
+    "for i in range(len(windows)):\n",
+    "    data = np.loadtxt(f'cv_values_window_{i}.txt')\n",
+    "    plt.hist(data[:,1])\n",
+    "    metafileline = f'cv_values_window_{i}.txt {windows[i]} 1000\\n'\n",
+    "    metafilelines.append(metafileline)\n",
+    "\n",
+    "plt.xlabel(\"r (nm)\")\n",
+    "plt.ylabel(\"count\")\n",
+    "\n",
+    "with open(\"metafile.txt\", \"w\") as f:\n",
+    "    f.writelines(metafilelines)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To compute the PMF we can use WHAM [[1](#References)]. An easy to use and widely compatible implementation is the WHAM program by Alan Grossfield which can be downloaded here: http://membrane.urmc.rochester.edu/?page_id=126. It is a C program so it will need to be compiled. The command below should work on Linux and Mac."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!wget http://membrane.urmc.rochester.edu/sites/default/files/wham/wham-release-2.0.11.tgz\n",
+    "!tar xf wham-release-2.0.11.tgz\n",
+    "!cd wham/wham && make"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To use `wham` we need a metadata file that lists the names of each CV timeseries file, the location of the harmonic restraints, and the value of the spring constant. We created this in the histogram plotting script earlier.\n",
+    " \n",
+    "The `wham` program is run using command line arguments. Read the documentation to find out more: http://membrane.urmc.rochester.edu/sites/default/files/wham/doc.pdf\n",
+    " \n",
+    "The command below will compute the PMF from our data. The command line arguments correspond to a range of 1.3nm to 3.3nm, 50 histogram bins, a tolerance of 1e-6, and a temperature of 300K.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!./wham/wham/wham 1.3 3.3 50 1e-6 300 0 metafile.txt pmf.txt > wham_log.txt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can then plot the computed PMF. It should look something like this:\n",
+    "![pmf.png](pmf.png)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# plot the PMF\n",
+    "pmf = np.loadtxt(\"pmf.txt\")\n",
+    "plt.plot(pmf[:,0], pmf[:,1])\n",
+    "plt.xlabel(\"r (nm)\")\n",
+    "plt.ylabel(\"PMF (kJ/mol)\")\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Next steps\n",
+    "\n",
+    "You should now perform error analysis to confirm the PMF you have calculated is an accurate representation of the system you are simulating.\n",
```

Comment:
Can you elaborate?  Even just a few sentences to summarize what sorts of analysis are helpful, or a link to a reference with more information.  Otherwise the reader may not know where to begin.